src/modules.cpp

```diff
@@ -884,6 +904,11 @@ BOOL WINAPI DetourRestoreAfterWithEx(_In_reads_bytes_(cbData) PVOID pvData,
         }
         VirtualProtect(pder->pidh, pder->cbidh, dwPermIdh, &dwIgnore);
     }
+	//Delete the payload after successful recovery to prevent repeated restore
+	if (fSucceeded)
+	{
+        DetourFreePayload(pder);
```

Comment:
Tabs for indentation here too.

Reply:
The latest commit has resolved it.

Reply:
I still see this on your most recent commit.

Reply:
@bgianfo, Do you mean "Tabs for indentation"? I checked the code again, but still couldn't find the place you mentioned. Can you point it out for me?
It may be possible to add a [.editorconfig ](https://docs.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2019) file to the project to help contributors reduce such formatting requirements

Reply:
@sonyps5201314 sure I can investigate adding that.